BaseClasses.py

```diff
@@ -262,12 +262,13 @@ def set_item_links(self):
             group["non_local_items"] = item_link["non_local_items"]
 
     # intended for unittests
-    def set_default_common_options(self):
-        for option_key, option in Options.common_options.items():
-            setattr(self, option_key, {player_id: option(option.default) for player_id in self.player_ids})
-        for option_key, option in Options.per_game_common_options.items():
-            setattr(self, option_key, {player_id: option(option.default) for player_id in self.player_ids})
+    @property
+    def default_common_options(self) -> Namespace:
         self.state = CollectionState(self)
```

Comment:
i think this line has to move somewhere else. cleanest would probably be to move it outside of the multiworld class (into the tests), since that's what generate/main currently does here: https://github.com/ArchipelagoMW/Archipelago/blob/cded105e6c881db3b76b39279e4fb1a6829fb123/Main.py#L76-L79

Reply:
Can we not just create CollectionState in the init? I don't know that it's possible, but it'd certainly solve this issue for both cases.